src/app/modules/submission/submission.controller.ts

```diff
@@ -344,7 +346,7 @@ export const streamEncryptedResponses: ControllerHandler<
       meta: logMeta,
       error: cursorResult.error,
     })
-
+    setErrorCode(cursorResult.error)
```

Comment:
I see that we already have the error codes automatically calling this on error initialisation, is this still required on controllers?

Reply:
Forgot to leave it out, thanks for the catch ![8df12dd](https://github.com/opengovsg/FormSG/pull/7443/commits/8df12ddf795667c7c89c6d51ad616f1b897425f6)